openc3/python/tester.py

```diff
@@ -0,0 +1 @@
+print('Hello World')
```

Comment:
This file is probably accidentally here?

Reply:
Will delete